content/get-haiku/release-notes-r1b2.md

```diff
@@ -0,0 +1,271 @@
+
+Additionally, there is now support for mice with more than three buttons.
+
+### DeskBar improvements
+
+The Deskbar has had various improvements. It now has a mini-mode, where the icons on the shelf expand leftwards. You can activate it by dragging the shelf space by the right handle to the top of the screen.
+
+<img src="/files/get-haiku/deskbar_minimode.png"/><br>
+<small><i>New mini-mode to put the Deskbar at the top of the screen.</i></small>
+
+Furthermore, there are more options for showing larger icons, which is especially useful on screens with a higher DPI.
+
+<img src="/files/get-haiku/deskbar_icon_sizes.png"/><br>
+<small><i>More size option for icons in DeskBar.</i></small>
+
+### Hardware Support: xHCI, hda, intel_extreme and NVMe
+
+Haiku's contributors have improved on xHCI, the host controller for USB3. This should make USB3 devices more stable. It also fixes some issues with booting Haiku from a USB3 storage device.
+
+Likewise, the Intel High Definition Audio (`hda`) driver has received some improvements, increasing the number of chipsets that it supports.
+
+The driver for Intel graphics chipsets (`intel_extreme`) has had various improvements. It supports more chipsets, as well as blacklists chipsets that are known not to work. The advantage to the latter is that a user will no longer have to manually disable the driver in the boot menu when their system uses one of the blacklisted chipsets.
+
+Finally, initial support for NVMe storage devices have been implemented. This is a protocol for modern SSD drives, which takes advantage of the properties of this modern storage medium, to improve I/O performance over previous storage protocols. Reminder: Haiku is still in beta quality, and there may be bugs in these storage drivers that cold cause data loss. 
+
+### Installation Improvements
+
+There are various improvements in the installation process. The first is that there is better support for EFI, though work remains to be done. Additionally, the Installer now supports optional packages that are included on the Installation medium. 
+
+DriveSetup will now display more information about the drives in your system. It will show the used space of existing partitions, and it is now also able to identify encrypted volumes.
+
+Finally, there is now also an upgrade path available from Haiku R1 beta 1 to beta 2.
+
+<img src="/files/get-haiku/installer_optionalpackages.png"/><br>
+<small><i>Optional package selection in the Installer.</i></small>
+
+### The Terminal now has an emulated Meta-key
+
+Certain early workstation keyboards included a key labeled "Meta" that functioned as an extra modifier key alongside Shift and Control. Its purpose was to give users an easy way of entering special characters: When held down it caused the eighth bit to be set on all characters read from the keyboard, allowing access to ​the "extended" portion of the ASCII character set (character codes above 127). 
```

Comment:
I would say this bit of UNIX history seems a bit irrelevant in the release notes. People who don't know what a Meta key is will not gain much from its support in Terminal, anyway

Reply:
Same though here.